sos/plugins/systemd.py

```diff
@@ -47,6 +48,13 @@ def setup(self):
         if self.get_option("verify"):
             self.add_cmd_output("journalctl --verify")
 
+        units = self.call_ext_prog("systemctl list-units -l")['output']
+        units = units.splitlines()
+
+        for unit in units[1:len(units)-7]:
```

Comment:
Nack. We can't rely on magic constants ("7 lines from the bottom"..) for parsing tool output - a change in systemd would break this.

It'd be better to define what you're expecting for a unit (e.g. two blank chars, followed by a name, followed by '.', followed by either 'timer', 'service' , etc.).